rotkehlchen/tests/unit/test_tasks_manager.py

```diff
@@ -548,19 +548,20 @@ def test_maybe_augmented_detect_new_spam_tokens(
 @pytest.mark.parametrize('number_of_eth_accounts', [0])
 def test_tasks_dont_schedule_if_no_eth_address(task_manager: TaskManager) -> None:
     """Test that we don't execute extra logic in tasks if no ethereum address is tracked"""
-    task_manager.should_schedule = True
-    for func in (
-        task_manager._maybe_update_yearn_vaults,
-        task_manager._maybe_update_ilk_cache,
-    ):
-        with (
-            patch('rotkehlchen.tasks.utils.should_run_periodic_task') as mocked_func,
+    with gevent.Timeout(5):  # this should not take long. Otherwise a long running task ran
```

Comment:
added a timeout to make test fail even if we change the way task manager's task `_maybe_update_yearn_vaults` and `_maybe_update_ilk_cache` works.